libtiff/tiff_file.py

```diff
@@ -235,10 +238,8 @@ def get_values(self, offset, typ, count):
             dtype = self.dtypes.type2dt.get(typ)
             bytes = type2bytes.get(typ)
             if dtype is None or bytes is None:
-                sys.stderr.write(
-                    'get_values: incomplete info for type=%r [%r]:'
-                    ' dtype=%s, bytes=%s\n' % (
-                        typ, ntyp, dtype, bytes))
+                warnings.warn('incomplete info for type=%r [%r]: dtype=%s, bytes=%s\n' % (
```

Comment:
This and the `warnings` import statement appears to be the only needed modification for resolving the issue.
Could you undo the rest of the changes?